### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 4

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#461] [Esami & Esoneri M-Z]
## Traccia

<img width="712" alt="image" src="https://github.com/user-attachments/assets/d4ed800e-6c1e-449c-b242-ecb5aa78a686">

**Comment by rimaout** [comment]
<img width="700" alt="image" src="https://github.com/user-attachments/assets/ddfd2dfd-8b3f-4ed5-bbab-7d6ae0cc8d1d">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/441c52c4-6067-472d-afdd-bf80b289ece3)
